forms/FormField.php

```diff
@@ -255,6 +251,22 @@ class FormField extends RequestHandler {
 	 */
 	protected $schemaData = [];
 
+	private static $casting = array(
+		'FieldHolder' => 'HTMLFragment',
+		'Field' => 'HTMLFragment',
```

Comment:
Because it doesn't always use renderTemplate(). This is a check to ensure it is forced to HTMLFragment in case subclasses don't.